lms/static/js/student_account/views/RegisterView.js

```diff
@@ -59,6 +59,8 @@
                     this.syncLearnerProfileData = data.thirdPartyAuth.syncLearnerProfileData || false;
                     this.errorMessage = data.thirdPartyAuth.errorMessage || '';
                     this.platformName = data.platformName;
+                    this.spaceName = data.spaceName;
```

Comment:
duplicate lines

Reply:
how so?

Reply:
@maneeshatejani1 Line 62 & Line 63 are same.

Reply:
got it